docs/process/style_guide.md

```diff
@@ -608,13 +608,12 @@ Main issue: [#113](https://github.com/unicode-org/icu4x/issues/113)
 
 When structs with public string fields contain strings, use the following type conventions:
 
-- If lifetime parameters are allowed, use `&'a str`.
-- If lifetime parameters are not allowed, use one of:
-  - [TinyStr](https://github.com/zbraniecki/tinystr) if the string is ASCII-only.
-  - [SmallString](https://crates.io/crates/smallstr) for shorter strings, with a stack size ∈ {8, 12, 16, 20} that fits at least 99% of cases.
-  - `Cow<'static, str>`  for longer strings.
+- `&'s str` if the struct does not need to own the string.
+- `Cow<'s, str>`  if the string could be borrowed or owned.
+- [TinyStr](https://github.com/zbraniecki/tinystr) if the string is ASCII-only.
+- [SmallString](https://crates.io/crates/smallstr) for shorter strings, with a stack size ∈ {8, 12, 16, 20} that fits at least 99% of cases.
```

Comment:
How did the `8, 12, 16, 20` numbers get determined? This could use the reason why these numbers were chosen.

How did the frequency of `99%` get determined? Is that supposed to be taken literally, or does it really mean "the vast majority of cases".